.github/workflows/deploy.yml

```diff
@@ -248,7 +249,7 @@ jobs:
 
   deploy-infra:
     needs: [begin-deployment]
-    uses: Enterprise-CMCS/managed-care-review/.github/workflows/deploy-infra-to-env.yml@main
+    uses: Enterprise-CMCS/managed-care-review/.github/workflows/deploy-infra-to-env.yml@mt-fix-lerna-7
```

Comment:
assuming this is deliberate and will get changed back to main in the future...

Reply:
This is something we have to do every time you make a change to one of the sub-workflows. Once this PR merges it will be followed up with a PR that switches things back to main since this will now be on main.